src/sestest/SoftwareEntitlementServer.cs

```diff
@@ -90,7 +92,7 @@ private void ConfigureKestrel(KestrelServerOptions serverOptions)
                 ServerCertificate = _options.ConnectionCertificate
             };
 
-            foreach (var address in addresses)
+            foreach (var address in addresses.Where(CanBind))
```

Comment:
Should this emit a warning message to the log if the address can't be bound?

(Thinking about the scenario where the user specifies an address on the command line and we then silently refuse to use it ...)

Reply:
The list of ports never originates with the user, so we don't need to log anything.